Adyen/Model/BalancePlatform/Address2.cs

```diff
@@ -0,0 +1,250 @@
+/*
+* Configuration API
+*
+*
+* The version of the OpenAPI document: 2
+* Contact: developer-experience@adyen.com
+*
+* NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+* https://openapi-generator.tech
+* Do not edit the class manually.
+*/
+
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.IO;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Text.RegularExpressions;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Newtonsoft.Json.Linq;
+using System.ComponentModel.DataAnnotations;
+
+namespace Adyen.Model.BalancePlatform
+{
+    /// <summary>
+    /// Address2
+    /// </summary>
+    [DataContract(Name = "Address-2")]
+    public partial class Address2 : IEquatable<Address2>, IValidatableObject
```

Comment:
Can we avoid this "-2" model?

Reply:
The devs need to go back and rename the model. it's not a breaking change for the API. we just need to know where & what to change